apps/dot-voting/app/components/VotingOption.js

```diff
@@ -60,11 +60,18 @@ const VotingOption = ({ valueSpring, label, percentage, color, threshold, userVo
             )}
           </div>
         )}
-        {percentage !== -1 &&
-            <Text size="xsmall" color="#98A0A2">
-              {Math.round(percentage)}%
+        <div>
+          {percentage !== -1 &&
+              <Text size="xsmall" color={theme.textTertiary}>
+                {Math.round(percentage)}%
+              </Text>
+          }
+          {allocation && (
```

Comment:
If you already pass either a string with parentheses or an empty string, you can remove this check. See my comment in `VotingOptions.js`.